.coveragerc

```diff
@@ -417,6 +417,7 @@ omit =
     homeassistant/components/insteon/utils.py
     homeassistant/components/incomfort/*
     homeassistant/components/intesishome/*
+    homeassistant/components/ipma/system_health.py
```

Comment:
This should have tests, as an example, you can have a look at <https://github.com/home-assistant/core/blob/dev/tests/components/hassio/test_system_health.py>